Mage/Task/BuiltIn/Symfony2/SymfonyAbstractTask.php

```diff
@@ -26,7 +29,13 @@ protected function getAppPath()
         } else {
             $defaultAppPath = 'app/console';
         }
-
-        return $this->getConfig()->general('symfony_app_path', $defaultAppPath);
+        
+        $path = '';
+        $niceness = intval($this->getParameter('niceness', 0));
```

Comment:
The parameter can be boolean value. We use Symfony's YAML component to parse configs' content, and it does support boolean values, as well as arrays, objects and `null`s.
So here you should write:
```php
$niceness = $this->getParameter('niceness', false);
if ($niceness) {
// ...
```